plugin.yaml

```diff
@@ -137,6 +138,18 @@ dsl_definitions:
         A boolean value, `true` or `false` whether
         to assume the user privileges.
       default: false
+    tags:
+      type: list
+      description: >
+        A list of tags, in order of desired execution. If these tags are provided, they will be called, and auto_tags will be ignored.
+      default: []
+    auto_tags:
```

Comment:
why actually we need this auto_tags? if the user wants to run the tasks that have tags so he will specify it, if not specified so we will run the playbook as before.

Reply:
Because users prefer to be lazy.